tap_snapchat_ads/streams.py

```diff
@@ -70,8 +70,7 @@
             # Reference: https://developers.snapchat.com/api/docs/?python#get-all-roles-in-organization
             'roles': {
                 'key_properties': ['id'],
-                'replication_method': 'INCREMENTAL',
-                'replication_keys': ['updated_at'],
+                'replication_method': 'FULL_TABLE', # made 'Roles' stream as FULL_TABLE stream, as we are not getting 'updated_at' from the API response
```

Comment:
Remove comments and add inline comments. Do we need to change integration tests?

Reply:
No, we need to update the integration tests after merging the Tap-Tester tests PR